### PR TITLE
Sokolovskyi/object instantiation

### PR DESCRIPTION
**Pull request by ssokolovskyi**

### CODE REVIEW

File: library/build.gradle [hunk 1]
```diff
@@ -18,17 +19,6 @@ android {
   }
 }
 
-apply plugin: 'android-unit-test'
```
Comment (chalup) [inline library/build.gradle]: This should be a separate commit that switches from some legacy custom plugin for unit tests to standard solution provided by android gradle plugin.

File: library/build.gradle [hunk 2]
```diff
@@ -40,9 +30,17 @@ dependencies {
...
+  testCompile "org.jetbrains.kotlin:kotlin-test-junit:$kotlin_version"
 
   compile 'com.android.support:support-annotations:19.1.0'
 }
 
 apply from: './gradle-mvn-push.gradle'
+
+project.afterEvaluate {
+  project.tasks.findByName("test").getTaskDependencies().getDependencies().each { task ->
+    task.jvmArgs('-noverify')
```
Comment (chalup) [inline library/build.gradle]: Not sure if that's still needed.

**Comment by ssokolovskyi** [comment]
@chalup updated
